pkg/remoteconfig/templates/avalanche-aliases.tmpl

```diff
@@ -0,0 +1,3 @@
+{
+  "{{.BlockChainID}}": ["{{.Alias}}"]
```

Comment:
pls make this support multiple aliases for multiple deploying blockchain at different times